packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -14,6 +14,8 @@ RUN dnf -y install dnf-plugins-core && \
 RUN dnf copr enable nmstate/ovs-el8 -y && \
     dnf copr enable networkmanager/NetworkManager-1.20 -y
 
+RUN dnf -y install https://extras.getpagespeed.com/release-el8-latest.rpm
```

Comment:
What is this?

Reply:
That is the repository used to fetch the python-dbus.There is no package available in Centos8 stream repository. Do you know any other way to get the package? Thanks!

Reply:
python3-dbus is in CentOS